subworkflows/icgc-argo-workflows/merg_sort_dup/main.nf

```diff
@@ -80,9 +81,16 @@ workflow MERG_SORT_DUP {
 
     ch_versions = ch_versions.mix(SAMTOOLS_MERGE.out.versions)
 
+    SAMTOOLS_SORT(
+        SAMTOOLS_MERGE.out.bam,
+        ref_org.fasta
+    )
+
+    ch_versions = ch_versions.mix(SAMTOOLS_SORT.out.versions)
+
```

Comment:
From looking around, it looks like we do not have to sort following merge.

https://stackoverflow.com/questions/47429140/is-sorting-necessary-for-merging-bam-files-using-bamtools
https://github.com/biod/sambamba/issues/266


Or it will merge and throw a warning if sorting is inconsistent between input:
https://www.biostars.org/p/251721/

GIven that we can leave sorting in the subworkflow prior or pass the alignment and do the read group aligned bam sorting.

If following the initial suggestion, we'll have to rename the workflow.